src/logger.js

```diff
@@ -0,0 +1,48 @@
+const chalk = require('chalk');
```

Comment:
I'm assuming this file is only ever run through node. if thats the case we could move it to the scripts folder fo clean up most includes.